docs/book/theme/css/custom.css

```diff
@@ -411,7 +411,11 @@ cite.literate-source > a::before {
 .tabset > input:nth-child(43):checked ~ .tab-panels > .tab-panel:nth-child(22),
 .tabset > input:nth-child(45):checked ~ .tab-panels > .tab-panel:nth-child(23),
 .tabset > input:nth-child(47):checked ~ .tab-panels > .tab-panel:nth-child(24),
-.tabset > input:nth-child(49):checked ~ .tab-panels > .tab-panel:nth-child(25){
+.tabset > input:nth-child(49):checked ~ .tab-panels > .tab-panel:nth-child(25),
+.tabset > input:nth-child(51):checked ~ .tab-panels > .tab-panel:nth-child(26),
+.tabset > input:nth-child(53):checked ~ .tab-panels > .tab-panel:nth-child(27),
+.tabset > input:nth-child(55):checked ~ .tab-panels > .tab-panel:nth-child(28),
+.tabset > input:nth-child(57):checked ~ .tab-panels > .tab-panel:nth-child(29){
```

Comment:
Because its not obvious to me: what exactly are these lines changing?

I checked the quickstart page and had not been able to find a relevant diff 🤔

Reply:
@mcbenjemaa ran into that issue when he added the provider for Proxmox. Basically it only makes sure that when you select a tab in the list of providers, the inner panel will be rendered. If the number of elements in this list is not sufficient, you will only see the tab but if you click on it, the panel will be empty.

So basically you have to make sure, that there are sufficient entries. I noticed that I was close to the limit of providers and wanted to add a few entries out of courtesy for future contributors.

Here is a related comment.
https://github.com/kubernetes-sigs/cluster-api/pull/9829#discussion_r1419044507

If you wanna test it, go to the docs folder, run `make serve`, remove entries from the CSS list to (47) and try to click on vSphere in the section `Initialization for common providers` and you will see an empty panel below.

Reply:
@sbueringer knows this